### ISSUE TITLE
NER and POS tagging transformers

### ISSUE THREAD

**Issue by xirect** [explosion/spacy-transformers#165]
I was wondering when the NER and POS transformers would be ready as it was once of the to-do's in your November 2019 talk.

**Comment by RnRoger** [comment]
I would also be interested in any kind of rough time estimation. We have to decide whether to wait for ner/pos transformers or use a different NLP pipeline altogether. We love SpaCy and want to use it but sadly time is a factor :cry:

**Comment by kohilin** [comment]
I found that I can obtain POS tags by manually adding the tagger pip in the original spaCy model to a transformer pipeline.

```
import spacy
nlp = spacy.load('en_trf_bertbaseuncased_lg')
doc = nlp("spaCy is awesome.")

# No tags 
print([(t.text, t.pos_) for t in doc]
>>> [('spaCy', ''), ('is', ''), ('awesome', ''), ('.', '')]

# Manual add
nlp_en = spacy.load('en_core_web_sm')
tagger_pipe = nlp_en.get_pipe("tagger")
nlp.add_pipe(tagger_pipe, after='sentencizer')

# Tags there
doc = nlp("spaCy is awesome.")
print([(t.text, t.pos_) for t in doc]
>>> [('spaCy', 'PROPN'), ('is', 'AUX'), ('awesome', 'ADJ'), ('.', 'PUNCT')]
```

If you mean "NER and POS **predicted** with Transformers",  please ignore this.

**Comment by xirect** [comment]
I am sorry, I meant that yeah...

**Comment by rogier-stegeman** [comment]
Any information on a date that the transformers will be ready?

**Comment by adrianeboyd** [comment]
Please try out `spacy-nightly` as a preview of how this is implemented in `spacy` v3.0 and `spacy-transformers` v1.0: https://nightly.spacy.io

**Comment by svlandeg** [comment]
I'm closing this, as this is now available with the new spaCy v3 - but do let us know if you run into any issues!
